apps/etterlatte-grunnlag/src/main/kotlin/grunnlag/GrunnlagService.kt

```diff
@@ -428,3 +509,45 @@ class LaastGrunnlagKanIkkeEndres(val behandlingId: UUID) :
             låst til en versjon av grunnlag (behandlingId=$behandlingId)
             """,
     )
+
+data class PersonopplysningerResponse(
+    val innsender: Personopplysning,
+    val soeker: Personopplysning,
+    val avdoede: List<Personopplysning>,
+    val gjenlevende: List<Personopplysning>,
+)
+
+data class Personopplysning(
+    val opplysningType: Opplysningstype,
+    val id: UUID,
+    val kilde: GenerellKilde,
+    val opplysning: PersonInformasjon,
+)
+
+data class GenerellKilde(
```

Comment:
Tanken med dette er da å støtte at tjenesten kan respondere med kilde av alle mulige typer uten krumspring med generics hverken i backend eller frontend. Synes det bør være mulig å kunne definere hva slags informasjon som må være tilstede for at noe er en _kilde_.